packages/moonstone/Switch/Switch.module.less

```diff
@@ -15,6 +15,7 @@
 	display: inline-block;
 	text-align: left;
 	cursor: default;
+	vertical-align: middle;
```

Comment:
@Djspaceg , What do you think about this?

Reply:
It was added to each element because we **don't want** to impose that visual rule at the UI level, affecting any theme using it. The alignment is decided by the theme; it's by no means a universal style. ToggleIcon does not have a stylesheet in Moonstone, so it was added to each consumer. It's mandatory for proper alignment regarding the styling of Moonstone.

Reply:
@ryanjduffy , Could you please check my patch and @Djspaceg 's opinion?

Reply:
Is a compromise to add a stylesheet to `moonstone/ToggleIcon` to apply the rule at an appropriate layer rather than duplicating it in multiple components? That would better leverage our infrastructure and avoid forgetting it in future changes/additions.

Reply:
Nevermind. Seems this was resolved otherwise.